src/activities/bargame/Bargame.qml

```diff
@@ -0,0 +1,388 @@
+                        height: rootWindow.height / (9 + (Activity.sublevel - 1) * Activity.elementSizeFactor[Activity.sublevel - 1])
+                        width: rootWindow.width / (15 + (Activity.sublevel - 1) * Activity.elementSizeFactor[Activity.sublevel - 1])
+                        visible: true
+                        // Numbering label
+                        GCText {
+                            text: index + 1
+                            fontSize: smallSize
+                            font.bold: true
+                            visible: ((index + 1) % 5 == 0 && index > 0) ? true : false
+                            anchors {
+                                horizontalCenter: parent.horizontalCenter
+                                bottom: parent.top
+                            }
+                        }
+                    }
+                }
+            }
+
+            // Hidden Answer Balls
+            Item {
+                id: answerBalls
+                // All green balls placement
+                Grid {
+                    id: answerBallsPlacement
+                    x: boxes.x
+                    y: boxes.y
+                    columns: 15
+                    rows: 1
+                    Repeater {
+                        model: answerBallsPlacement.columns
+                        Image {
+                            source: "qrc:/gcompris/src/activities/bargame/resource/green_ball.svg"
+                            height: rootWindow.height / (9 + (Activity.sublevel - 1) * Activity.elementSizeFactor[Activity.sublevel - 1])
+                            width: rootWindow.width / (15 + (Activity.sublevel - 1) * Activity.elementSizeFactor[Activity.sublevel - 1])
+                            opacity: 0.0
+                        }
+                    }
+                }
+            }
+
+            // Masks
+            Grid {
+                id: masks
+                x: boxes.x
+                y: boxes.y
+                rows: 1
+                columns: 15
+                Repeater {
+                    id: startMask
+                    model: masks.columns
+                    Image {
+                        id: greenMask
+                        source: Activity.url + ((index == boxes.columns - 1) ? "mask_last.svg" : "mask.svg")
+                        height: rootWindow.height / (9 + (Activity.sublevel - 1) * Activity.elementSizeFactor[Activity.sublevel - 1])
+                        width: rootWindow.width / (15 + (Activity.sublevel - 1) * Activity.elementSizeFactor[Activity.sublevel - 1])
+                    }
+                }
+            }
+        }
+
+        // OK BUTTON
+        Image {
+            id: playLabel
+            x: rootWindow.width / 1.2
+            y: rootWindow.height / 2.4
+            width: rootWindow.width / 20
+            height: width
+            source: "qrc:/gcompris/src/activities/bargame/resource/ok.svg"
```

Comment:
this ok button does not render well, you can use the one in core (bar_ok.svg) for now (I've created a OKButton.qml but not yet pushed it so we'll updated it later if needed)